Receiving warnings when building from source in R 3.3.1

I'm seeing warnings during installing cyclops (as a dependency of CohortMethod) from source using the following command:

`
install_git("OHDSI\CohortMethod")
`

The warning is:
```text
RcppCyclopsInterface.cpp: In member function 'bsccs::priors::JointPriorPtr bsccs::RcppCcdInterface::makePrior(const std::vector<std::basic_string<char> >&, const std::vector<double>&, const ProfileVector&, const HierarchicalChildMap&, const NeighborhoodMap&)':
RcppCyclopsInterface.cpp:581:33: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
         && basePriorName.size() == length
                                 ^
RcppCyclopsInterface.cpp:582:32: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
         && baseVariance.size() == length) {
                                ^
````

I think the warning is harmless but could this be cleaned up?

-Chris

--- a/src/RcppCyclopsInterface.cpp
+++ b/src/RcppCyclopsInterface.cpp
@@ -573,7 +573,7 @@ priors::JointPriorPtr RcppCcdInterface::makePrior(const std::vector<std::string>
 		const ProfileVector& flatPrior, const HierarchicalChildMap& hierarchyMap, const NeighborhoodMap& neighborhood) {
 	using namespace bsccs::priors;
 
-    const int length = modelData->getNumberOfColumns();
+    const size_t length = modelData->getNumberOfColumns();
 
  	if (   flatPrior.size() == 0
  	    && hierarchyMap.size() == 0
@@ -584,7 +584,7 @@ priors::JointPriorPtr RcppCcdInterface::makePrior(const std::vector<std::string>
         auto first = bsccs::priors::CovariatePrior::makePrior(parsePriorType(basePriorName[0]), baseVariance[0]);
         auto prior = bsccs::make_shared<MixtureJointPrior>(first, length);
 
-        for (int i = 1; i < length; ++i) {
+        for (size_t i = 1; i < length; ++i) {
             auto columnPrior = bsccs::priors::CovariatePrior::makePrior(parsePriorType(basePriorName[i]), baseVariance[i]);
             prior->changePrior(columnPrior, i);
         }

